<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndustryDetailDialog</class>
 <widget class="QDialog" name="IndustryDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Industry Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>878</width>
        <height>644</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="basicInfoGroup">
         <property name="title">
          <string>Industry information</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelName">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="name_edit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelLocalName">
            <property name="text">
             <string>Local Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="local_name_edit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSymbol">
            <property name="text">
             <string>Track Symbol:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="symbol_edit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelProcessBlocks">
            <property name="text">
             <string>Process in Blocks:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="process_blocks_check">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="tracksGroup">
         <property name="title">
          <string>Tracks</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableWidget" name="tracks_table">
            <property name="columnCount">
             <number>3</number>
            </property>
            <column>
             <property name="text">
              <string>Route Prefix</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Track Section</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Track Direction</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="trackButtonLayout">
            <item>
             <widget class="QPushButton" name="remove_track_button">
              <property name="text">
               <string>Remove Selected Track</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="trackButtonSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="producersGroup">
         <property name="title">
          <string>Incoming cars</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableWidget" name="producers_table">
            <property name="columnCount">
             <number>6</number>
            </property>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Car Type</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Processed Tags</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Capacity</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Hours</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Produces</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
